sdlib/d/gc/page.d

```diff
@@ -264,6 +274,9 @@ private:
 		}
 
 		auto n = block.reserve(pages, dirty);
+
+		usedPageCount += pages;
+
```

Comment:
Same as for allocHugeImpl this is right in between the block operation. Just move it before the reserve (or after the whole thing).